$NetBSD: patch-glib_meson.build,v 1.2 2019/06/16 01:54:40 ryoon Exp $

- Don't bother with gtester as it's slated for deprecation anyway:
  https://gitlab.gnome.org/GNOME/glib/issues/1441
- Don't build tests

--- glib/meson.build.orig	2019-06-10 17:47:20.000000000 +0000
+++ glib/meson.build
@@ -284,7 +284,7 @@ libglib_dep = declare_dependency(
   include_directories : [configinc, glibinc])
 
 pkg.generate(libglib,
-  libraries : [libintl],
+  libraries : [libintl, '-Wl,-rpath${libdir}'],
   libraries_private : [osx_ldflags, win32_ldflags],
   subdirs : ['glib-2.0'],
   extra_cflags : ['-I${libdir}/glib-2.0/include'] + win32_cflags,
@@ -324,24 +324,8 @@ if host_system == 'windows'
       include_directories : configinc,
       dependencies : [libglib_dep])
   endif
-else
-  gtester = executable('gtester', 'gtester.c',
-    install : true,
-    include_directories : configinc,
-    dependencies : [libglib_dep])
 endif
 
-report_conf = configuration_data()
-report_conf.set('GLIB_VERSION', glib_version)
-report_conf.set('PYTHON', python_name)
-configure_file(
-  input: 'gtester-report.in',
-  output: 'gtester-report',
-  install_dir: get_option('bindir'),
-  configuration: report_conf,
-  install_mode: 'rwxr-xr-x'
-)
-
 install_data('glib_gdb.py', install_dir : join_paths(glib_pkgdatadir, 'gdb'))
 
 gdb_conf = configuration_data()
@@ -370,5 +354,3 @@ if enable_systemtap
     install_dir : tapset_install_dir,
     install : true)
 endif
-
-subdir('tests')
